modules/swagger-codegen/src/main/resources/go/README.mustache

```diff
@@ -38,25 +38,49 @@ Class | Method | HTTP request | Description
 {{/model}}{{/models}}
 
 ## Documentation For Authorization
-
-{{^authMethods}} All endpoints do not require authorization.
+{{^authMethods}} Endpoints do not require authorization.
 {{/authMethods}}{{#authMethods}}{{#last}} Authentication schemes defined for the API:{{/last}}{{/authMethods}}
-{{#authMethods}}## {{{name}}}
-
+{{#authMethods}}
+## {{{name}}}
 {{#isApiKey}}- **Type**: API key 
-- **API key parameter name**: {{{keyParamName}}}
-- **Location**: {{#isKeyInQuery}}URL query string{{/isKeyInQuery}}{{#isKeyInHeader}}HTTP header{{/isKeyInHeader}}
+!!! NOT IMPLEMENTED !!!
```

Comment:
I recall how I became confused here...

API Key in Query String: https://github.com/antihax/swagger-codegen/blob/New-GoClient-GoServer/samples/client/petstore/go/go-petstore/pet_api.go#L99

I will add the header equivalent tonight and alter the documentation.